lsp-completion.el

```diff
@@ -67,17 +67,6 @@ ignored."
   :group 'lsp-completion
   :package-version '(lsp-mode . "7.0.1"))
 
-(defcustom lsp-completion-show-detail t
```

Comment:
I would prefer to keep these options as it has been default for ages and works well for most of other language servers.
Also, people can prefer to not show any of these information on the candidate's annotation as well, we shouldn't take that away.